manifests/l/LWorks/SuperEggHunt/1.53/LWorks.SuperEggHunt.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: LWorks.SuperEggHunt
+PackageVersion: 1.53
+PackageLocale: en-US
+Publisher: LWorks
+PackageName: Super Egg Hunt
+License: Copyright LWorks 2002 - 2022

```

A copyright is not a license; I believe they use a proprietary one, but I'm not certain
```suggestion
License: Proprietary
Copyright: Copyright LWorks 2002 - 2022
```